tests/test_register_webhook.py

```diff
@@ -54,7 +48,7 @@ def test_register_webhook_request_exception(
         channels=["zaken"],
     )
 
-    with patch("ape_pie.APIClient.post", side_effect=RequestException("exception")):
+    with patch("requests.Session.post", side_effect=RequestException("exception")):
```

Comment:
requests_mock can throw exceptions and return status codes, so you could use that here